ci-operator/step-registry/cucushift/installer/check/ibmcloud/kms-key/cucushift-installer-check-ibmcloud-kms-key-commands.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# IBM Cloud CLI login
+function ibmcloud_login {
+    export IBMCLOUD_CLI=ibmcloud
+    export IBMCLOUD_HOME=/output
+    region="${LEASED_RESOURCE}"
+    rg=$1
+    export region
+    "${IBMCLOUD_CLI}" config --check-version=false
+    echo "Try to login to ${rg}..."
+    "${IBMCLOUD_CLI}" login -r ${region} -g ${rg} --apikey @"${CLUSTER_PROFILE_DIR}/ibmcloud-api-key"
+}
+
+RESOURCE_GROUP=$(jq -r .ibmcloud.resourceGroupName ${SHARED_DIR}/metadata.json)
+
+ibmcloud_login ${RESOURCE_GROUP}
+
+critical_check_result=0
+
+kp_file=${SHARED_DIR}/ibm_kp.json
+
+if [ -f ${kp_file} ]; then
+    id=$(jq -r .id ${kp_file})
+else
+    echo "[ERROR] fail to get the key protect instance id !!"
+    exit 1
+fi
+
+#check the kpKey whether used in the volumes of the nodes(master & worker)
+mapfile -t vols < <(ibmcloud kp registrations -i ${id} -o JSON  | jq -r .[].resourceCrn)
+echo "INFO: key registrations list is: ${#vols[@]}" "${vols[@]}"
+
+#check that node os disk is encrypted
+machines=$(oc get machines -A --no-headers | awk '{print $2}')
+
+for machine in ${machines}; do
+    echo "--- check machine ${machine} ---"
+    volCrn=$(ibmcloud is instance ${machine} --output JSON | jq -r .boot_volume_attachment.volume.crn)
+    volCrn="\<${volCrn}\>"
```

Comment:
I run some local testing, it works well, does not need `\<` stuff, do I miss something?
```
[jianlinliu@jialiu-laptop installer]$ vols=("crn:v1:bluemix:public:is:jp-tok-1:a/fdc2e14cf8bc4d53a67f972dc2e2c861::volume:r022-36368193-b227-4ce9-b3ee-86d06670ae6f" "crn:v1:bluemix:public:is:jp-tok-1:a/fdc2e14cf8bc4d53a67f972dc2e2c861::volume:r022-4fc2eead-81f9-44f3-88ae-a32f34e96dcc")
[jianlinliu@jialiu-laptop installer]$ volCrn="crn:v1:bluemix:public:is:jp-tok-1:a/fdc2e14cf8bc4d53a67f972dc2e2c861::volume:r022-36368193-b227-4ce9-b3ee-86d06670ae6f"
[jianlinliu@jialiu-laptop installer]$ #shellcheck disable=SC2076
[jianlinliu@jialiu-laptop installer]$ if [[ ! " ${vols[*]} " =~ " ${volCrn} " ]]; then echo bad; fi
[jianlinliu@jialiu-laptop installer]$ volCrn="adadafd"
[jianlinliu@jialiu-laptop installer]$ if [[ ! " ${vols[*]} " =~ " ${volCrn} " ]]; then echo bad; fi
bad

```

Reply:
in this test, need not volCrn="\<${volCrn}\>"
if volCrn is "v1 v2", need add \<